.rubocop.yml

```diff
@@ -0,0 +1,26 @@
+inherit_from: .rubocop_todo.yml
+
+AllCops:
+  Exclude:
+    - vendor/**/*
+    - gemfiles/**/*
+
+Metrics/LineLength:
+  Max: 140
+
+# Can't do without a little table-flip there ;)
+Style/AsciiComments:
+  Exclude:
+    - 'test/scope_test.rb'
```

Comment:
I excluded it like this (you can do line-based enabling/disabling of cops, but that would look weird), so you can keep the table flip emoticon, but I think in general it's a good rule. Do you agree?